docs/2.0/03-inputs-en.md

```diff
@@ -114,19 +114,32 @@ location = "Local"
 
 #
 # The table to be scanned
-# Required
+# Optional. If not specified, output router will be used.
 [[input.config.table-configs]]
 schema = "test_1"
 table = "test_source_*"
 
 [[input.config.table-configs]]
 schema = "test_2"
 table = "test_source_*"
+
 # Optional
 # Enforce the column to scan for tables. If you don't specifiy this value, the system will use a column that has unique index.
 # Make sure you know what you are doing: YOU NEED TO SPECIFY A COLUMN THAT HAVE A UNIQUE INDEX.
 scan-column = "id"
 
+# you can ignore talbes defined in input.config.table-configs
+# gravity do not support scan of these tables if it matches all of these condition:
```

Comment:
this statement is not related to `ignore-tables` config, it belongs to the whole mysql batch input.